llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3667,9 +3669,8 @@ void LoopVectorizationCostModel::collectLoopUniforms(ElementCount VF) {
 
       // ExtractValue instructions must be uniform, because the operands are
       // known to be loop-invariant.
-      if (auto *EVI = dyn_cast<ExtractValueInst>(&I)) {
-        assert(IsOutOfScope(EVI->getAggregateOperand()) &&
-               "Expected aggregate value to be loop invariant");
+      if (auto *EVI = dyn_cast<ExtractValueInst>(&I);
```

Comment:
Perhaps this is just personal opinion, but I'm not really keen on multi-statement if conditions as I find it a bit confusing to read. I haven't seen other examples this elsewhere in LLVM, although I could be wrong! I thought it was more in keeping with LLVM style to write something like:

```
  auto *EVI = dyn_cast<ExtractValueInst>(&I);
  if (EVI && IsOutOfScope(EVI->getAggregateOperand()))
```
?

Reply:
Happy to change this :+1: I think I may have picked up this style from another project :slightly_smiling_face: